cdisc_rules_engine/sql_operations/sql_base_operation.py

```diff
@@ -207,6 +209,68 @@ def _get_variables_metadata_from_standard_model(self, domain: str) -> List[dict]
         variables_metadata = self.params.standards_context.get_domain_variables(domain)
         return variables_metadata
 
+    def _get_variables_metadata_from_standard_model(self, domain: str) -> List[dict]:
+
+        # Need to investigate to understand difference between _get_variables_metadata_from_standard and
+        # _get_variables_metadata_from_standard_model - they appear to be doing very similar things
+        # and yet the output of each is slightly different
+
+        # Also need to think about where to put this - to what extent are standards and
+        # models correlated?
+
+        # For now, reverting to previous implementation of _get_variables_metadata_from_standard_model
+        # and keeping implementation of _get_variables_metadata_from_standard
+
+        # Will look to rewrite this function to be less standard-dependent soon
+
+        # For SQL operations, use a simplified version that works with available metadata
+        model_details = self.params.standards_context.get_model_metadata()
+
+        # Handle SUPP domain normalization like the original function
+        if domain and (domain.upper().startswith("SUPP") or domain.upper().startswith("SQ")) and len(domain) > 2:
```

Comment:
Lots of the logic in here is standard specific, so should be in the standards context. Anything which references SUPP, anything which calls sdtm_utilities, they're all standard specific